### PR TITLE
Complete skeleton of home

### PR DESCRIPTION
**Pull request by ericong18**
## Description
This PR focuses on getting the skeleton set up for the home listings page.

### CODE REVIEW

**Comment by jshin029** [comment]
@ericong18  Hey Eric can you fix the merge conflicts first before the pr is reviewed?

**Comment by ericong18** [comment]
@jshin029 i fixed

File: frontend/src/App.js [hunk 1]
```diff
@@ -14,6 +15,7 @@ function App() {
           <Route exact path="/" component={Landing} />
           <Route exact path="/login" component={Login} />
           <Route exact path="/signup" component={Signup} />
+          <Route path="/home" component={Home} />
```
Comment (jshin029) [inline frontend/src/App.js]: can we keep it consistent and use exact here as well?

Comment (ericong18) [reply frontend/src/App.js]: @jshin029 due to how the `Home` component will be implemented in the future, let's not

**Pull request merged**